packages/google-pay-integration/jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    displayName: 'google-pay-integration',
+    preset: '../../jest.preset.js',
+    globals: {
+        'ts-jest': {
+            tsconfig: '<rootDir>/tsconfig.spec.json',
+            diagnostics: false,
+        }
+    },
+    transform: {
```

Comment:
Sorry missed this one in the previous one. We will need to document this part till we have out own generator to this work.

Reply:
Sorry I forgot to take it out as well. As you suggested, it's now in the `preset` so it doesn't need to be redefined each time.